vscode-lean4/test/suite/info/info.test.ts

```diff
@@ -0,0 +1,32 @@
+import * as assert from 'assert';
+import { suite } from 'mocha';
+import * as vscode from 'vscode';
+import { initLean4Untitled, assertStringInInfoview, findWord } from '../utils/helpers';
+import { InfoProvider } from '../../../src/infoview';
+
+suite('InfoView Test Suite', () => {
+
+    test('Copy to Comment', async () => {
+
+        console.log('=================== Copy to Comment ===================');
+
+        const lean = await initLean4Untitled('#eval Lean.versionString');
+        const info = lean.exports.infoProvider as InfoProvider;
+
+        await assertStringInInfoview(info, '4.0.0-nightly-');
+
+        console.log('Clicking copyToComment button in InfoView');
+        await info.runTestScript('document.getElementsByClassName(\'codicon-quote\')[0].click()');
```

Comment:
Please just give the button its own class name, and then access it using that name.  I don't want this test to fail just because we change the button icon.

Reply:
Done.